src/DoctrineORMModule/Service/DBALConnectionFactory.php

```diff
@@ -60,13 +60,16 @@ public function __invoke(ContainerInterface $container, $requestedName, array $o
         $eventManager  = $container->get($options->getEventManager());
 
         $connection = DriverManager::getConnection($params, $configuration, $eventManager);
-        $platform = $connection->getDatabasePlatform();
-        foreach ($options->getDoctrineTypeMappings() as $dbType => $doctrineType) {
-            $platform->registerDoctrineTypeMapping($dbType, $doctrineType);
```

Comment:
A simpler fix is to simply this change this line to `$connection->getDatabasePlatform()->registerDoctrineTypeMapping($dbType, $doctrineType);`